cpu-hog/prow_run.sh

```diff
@@ -15,19 +15,14 @@ SCENARIO_FOLDER="$krkn_loc/scenarios/arcaflow/cpu-hog"
 echo "Printing cluster version"
 oc version
 
-# Copy config to kraken
-cp node-cpu-hog/input.yaml.template $SCENARIO_FOLDER/input.yaml.template
-
 source node-cpu-hog/env.sh
-source env.sh
```

Comment:
Think we need this file

Reply:
We are already sourcing the env.sh file in https://github.com/redhat-chaos/prow-scripts/blob/main/cpu-hog/prow_run.sh#L8